test.js

```diff
@@ -0,0 +1,78 @@
+require( 'sugar' );
+var assert = require( 'assert' );
+var Promise = require( 'bluebird' );
+var parseBig = require( './index' );
+
+var MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER;
+var MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER;
+
+describe( 'Parse Big', function () {
+    it( 'error', function ( done ) {
+        var badJsonString = '{"name":"somename}';
+
+        Promise.resolve( badJsonString )
```

Comment:
Not sure why you need `Promise`/`bluebird` for code that is completely synchronous... seems like a big dependency to add just for calling `.then` (which isn't necessary for synch code)